ClavaLaraExtraApi/test/pt/up/fe/specs/clava/extra/ClavaMathExtraTest.java

```diff
@@ -1,43 +0,0 @@
-/**
```

Comment:
Keep file

Reply:
Resolved in 44508bf26 with the creation of the `ClavaLaraExtraAPIs` branch.